WorkloadManagementSystem/Service/WMSAdministratorHandler.py

```diff
@@ -194,105 +162,6 @@ def export_getSiteMaskSummary(cls):
 
     return S_OK(siteDict)
 
-  ##############################################################################
-  types_getCurrentPilotCounters = [dict]
-
-  @classmethod
-  def export_getCurrentPilotCounters(cls, attrDict={}):
```

Comment:
Maybe I am mistaken, but I guess that for backward compatibility, you can't just remove all these methods at the same time you introduce the new service ?